benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1168,6 +1168,12 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "showPostponementDetailsPage", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "postponementPreviewDocument", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "unprocessedPostponementRequest", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "actionPostponementRequestSelected", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "actionPostponementRequestSelected", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "actionPostponementRequestSelected", "UserRole": "caseworker-sscs-judge", "CRUD": "CRUD"},
```

Comment:
Should caseworker-sscs-judge have the permission for this? They don't have access to the event where this is used.

Reply:
Done